### PR TITLE
Get class mirror from test suite

### PR DESCRIPTION
**Pull request by junmendoza**
@aparajit-pratap PTAL
This is getting the class mirror from the test suite

### CODE REVIEW

File: src/Engine/ProtoScript/Runners/LiveRunner.cs [hunk 1]
```diff
@@ -1172,7 +1172,7 @@ private void InitCore()
             runnerCore.Options.IncludeDirectories = configuration.SearchDirectories.ToList();
             foreach (var item in configuration.PassThroughConfiguration)
             {
-                runtimeCore.DSExecutable.RuntimeData.Configurations[item.Key] = item.Value;
+                runnerCore.Configurations[item.Key] = item.Value;
             }
```
Comment (aparajit-pratap) [inline src/Engine/ProtoScript/Runners/LiveRunner.cs]: Could you stick this into a separate PR as discussed as it is related to fixing a separate crash.

File: test/DynamoCoreTests/DSEvaluationViewModelTest.cs [hunk 2]
```diff
@@ -115,8 +115,11 @@ public void AssertClassName(string guid, string className)
             string varname = GetVarName(guid);
             var mirror = GetRuntimeMirror(varname);
             Assert.IsNotNull(mirror);
```
Comment (aparajit-pratap) [inline test/DynamoCoreTests/DSEvaluationViewModelTest.cs]: I'm not sure why we are checking for the runtime mirror obtained here from the `guid` in the first place. This check does not go with the purpose of this function, which is to verify the class name. Is it possible to refactor the `guid` check into another API?

**Comment by aparajit-pratap** [comment]
Now that you've merged https://github.com/DynamoDS/Dynamo/pull/3877, you would need to sync this branch to make it up-to-date.

File: src/Engine/ProtoScript/Runners/LiveRunner.cs (hunk 1 above)
Comment (junmendoza) [inline src/Engine/ProtoScript/Runners/LiveRunner.cs]: Done

File: test/DynamoCoreTests/DSEvaluationViewModelTest.cs (hunk 2 above)
Comment (junmendoza) [inline test/DynamoCoreTests/DSEvaluationViewModelTest.cs]: The RuntimeMirror check was alsways there, i guess it was to get the varname value

Comment (aparajit-pratap) [inline test/DynamoCoreTests/DSEvaluationViewModelTest.cs]: Yeah I know, I was asking you to move it out of here or get rid of it if it's not required.

**Pull request merged**